python/erdos/operators/window.py

```diff
@@ -0,0 +1,78 @@
+"""
+   Windowing Operators will collect incoming inputs into groups.
+   Messages sent from these operators take the form of: 
+   (most recent message's timestamp, list of Messages (msg.timestamp, msg.data) ).
+"""

```

Comment:
This is good. Nitpicking: "Windowing operators collect incoming messages into groups."
Then maybe to be slightly clearer "Messages sent from these operators contain a list of Messages as data and have a timestamp corresponding to the most recent Message in the list."

Also, can you reformat the docstring according to the [Google styleguide](https://google.github.io/styleguide/pyguide.html#382-modules)?